docker/README.md

```diff
@@ -172,3 +188,27 @@ export GZ_VERSION=garden
 
 For details, see Gazebo documentation on
 [Installing Gazebo with ROS](https://gazebosim.org/docs/garden/ros_installation).
+
+## For maintainers
+
+### For maintainers only: Save Docker image to disk
+
+You may want to save the compiled Docker image to disk, e.g. on a USB drive.
+
+This will take a few minutes (gzip option to internal SSD took 5 minutes on
+Dell XPS 15 9570 from 2018).
+
+You can use gzip (file size much smaller than tar):
+```
+docker save osrf/icra2023_ros2_gz_tutorial:tutorial_nvidia | gzip > tutorial_nvidia.tar.gz
+docker save osrf/icra2023_ros2_gz_tutorial:tutorial_no_nvidia | gzip > tutorial_no_nvidia.tar.gz
+```
+
+Regular tar (larger file size):
+```
+docker save osrf/icra2023_ros2_gz_tutorial:tutorial_nvidia > tutorial_nvidia.tar
+docker save osrf/icra2023_ros2_gz_tutorial:tutorial_no_nvidia > tutorial_no_nvidia.tar
+```
```

Comment:
I don't think we particularly need to have this option; the compressed one is better for almost all purposes, and is supported by `docker load`.

Reply:
I'll remove it. I included it because docker has it in their documentation, so I wasn't sure if some operating systems don't come with gzip. Looks like Ubuntu comes with it. macOS I can't tell for sure; it comes with `tar` with the -z option, but maybe doesn't come with `gzip`... I don't know the difference. Windows 10 does since 2018 (version 1803 build 17063).